lucet-runtime/lucet-runtime-internals/src/alloc/tests.rs

```diff
@@ -964,4 +1013,11 @@ macro_rules! alloc_tests {
 }
 
 #[cfg(test)]
-alloc_tests!(crate::region::mmap::MmapRegion);
+mod mmap {
```

Comment:
Wait, did I miss the alloc tests in my conversion??

Reply:
Yeah, though since they're part of `internals` rather than `lucet-runtime-tests`, I don't blame you for missing them.